examples/fargate-serverless/README.md

```diff
@@ -33,43 +36,171 @@ Enter `yes` at command prompt to apply
 
 The following command will update the `kubeconfig` on your local machine and allow you to interact with your EKS Cluster using `kubectl` to validate the CoreDNS deployment for Fargate.
 
-1. Run `update-kubeconfig` command:
+1. Check the Terraform provided Output, to update your `kubeconfig`
+
+```hcl
+Apply complete! Resources: 63 added, 0 changed, 0 destroyed.
+
+Outputs:
+
+configure_kubectl = "aws eks --region us-west-2 update-kubeconfig --name fully-private-cluster"
+```
+
+2. Run `update-kubeconfig` command, using the Terraform provided Output, replace with your `$AWS_REGION` and your `$CLUSTER_NAME` variables.
+
+```sh
+aws eks --region <$AWS_REGION> update-kubeconfig --name <$CLUSTER_NAME>
+```
+
+3. Validate if the Fargate Profiles were successfully created in the Cluster.
```

Comment:
Not valid - we don't need to check these

Reply:
OK. Done!